tests/observability/metrics/conftest.py

```diff
@@ -1046,3 +1047,23 @@ def deleted_ssp_operator_pod(admin_client, hco_namespace):
 @pytest.fixture(scope="class")
 def initiate_metric_value(request, prometheus):
     return get_metrics_value(prometheus=prometheus, metrics_name=request.param)
+
+
+@pytest.fixture()
+def vm_for_vm_disk_allocation_size_test(namespace, admin_client):
+    vm_name = "vm-disk-metric-allocation-test"
+    with create_cirros_vm(
+        storage_class=StorageClassNames.CEPH_RBD,
```

Comment:
Please don't use storage class names. You can use `py_config["default_storage_class"]`